src/pybind/mgr/cephadm/module.py

```diff
@@ -435,6 +435,36 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
             default=True,
             desc='Pass --cgroups=split when cephadm creates containers (currently podman only)'
         ),
+        Option(
+            'prometheus_web_user',
+            type='str',
+            default='admin',
+            desc='Prometheus web user'
+        ),
+        Option(
+            'prometheus_web_password',
+            type='str',
+            default='admin',
+            desc='Prometheus web password'
+        ),
+        Option(
+            'alertmanager_web_user',
+            type='str',
+            default='admin',
+            desc='Alertmanager web user'
+        ),
+        Option(
+            'alertmanager_web_password',
+            type='str',
+            default='admin',
+            desc='Alertmanager web password'
+        ),
```

Comment:
> is it necessary these be module config options rather than fields in the relevant service specs? Intuitively I feel like that's where these prometheus/alertmanager specific things would belong.

Well, these parameters will be moved to the mon-store and we don't want the user/password to be exposed as module options. I'm just keeping them as conf options in this first PR version so later I can easy track and move them to mon-store.